webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -25,27 +25,29 @@ const getNotifications = url => (dispatch) => {
 
   if (isDocumentVisible) {
     API.get(url)
-      .done(onGetNotificationsSuccess)
-      .fail(onGetNotificationsFailed)
-      .always(triggerPolling);
+      .then(onGetNotificationsSuccess)
+      .catch(onGetNotificationsFailed);
   } else {
     // document is not visible, keep polling without api call
     triggerPolling();
   }
 
-  function onGetNotificationsSuccess(response) {
+  function onGetNotificationsSuccess({ data }) {
     dispatch({
       type: NOTIFICATIONS_GET_NOTIFICATIONS,
       payload: {
-        notifications: response.notifications,
+        notifications: data.notifications,
       },
     });
+    triggerPolling();
```

Comment:
why did you add it here too?

Reply:
there is no a special call back for always / done, though we could use `then` after catch.